### ISSUE TITLE
Next Step Microgrant: Low Latency Cache Layer For IPFS

### ISSUE THREAD

**Issue by MihanixA** [ipfs/devgrants#132]
### 1. What is your project? (max 100 words)
The idea is to create an L2-like solution over IPFS that would grant lower read latency by using SSD or RAM storage devices. I've heard a lot of ides for the applications over IPFS that require 'hot data' like real-time games or etc, though current read latency is not acceptable for those.

### 2. How are you planning to improve this project? (max 200 words)
I am going to write an MVP that contains
1. Node (ran by users). It subscribes data feed and stores 'hot data' in-memory. It also registers request latency, allocated storage and calculates score that node host has earned.
2. Router server.
 
### 3. Will the work be Open Source?
MIT for sure.

### 4. If selected, do you agree to complete weekly updates and a grant report upon conclusion?
Yep. I'd also share my Notion/Trello and other project-related data.
  
### 5. Does your proposal comply with our Community Code of Conduct?
Too long, though read. Yep.

### 6. Links and submissions

  
* Have you submitted to the IPFS Community Showcase ([instructions](https://github.com/ipfs/community/blob/master/README.md#showcase-your-project))? 
Not yet.
  

* Have you added your project to the [IPFS Ecosystem Directory](https://airtable.com/shrjwvk9pAeAk0Ci7)? Yes
  
* If your project began at a hackathon, have you submitted it for the relevant Protocol Labs prizes? Include links here if available:
  
* Have you filled out the [Interplanetary Builder Feedback Survey](https://airtable.com/shrDZMizx03jOa4mQ)? Yes
  
### Additional questions:
* For each team member(s), please list name, email, Github account, and role in the project. Sitnikov Mikhail https://github.com/MihanixA CEO/CTO

**Comment by SeedingTrees** [comment]
Thank you for the application.  We will need to see some progress on this project before we can review your application.  Please let us know once you have a repo or demo that we can view.  For now, we will pause on reviewing this application.

**Comment by ErinOCon** [comment]
Hi @MihanixA, I hope you are doing well! Do you anticipate providing repo or demo updates in this feed? Please do not hesitate to be in touch with updates or questions.

**Comment by ErinOCon** [comment]
@MihanixA, circling back to confirm if a repo or demo will be available soon. Thanks!

**Comment by ErinOCon** [comment]
Hi @MihanixA, this issue has been closed. Please feel welcome to open a new issue at any time.
